### PR TITLE
explorer landing page update

### PR DESCRIPTION
**Pull request by Jibz1**
- Transaction page done 
- Set transaction page to landing page. The home current home page can be accessed from `home-other`
- Set added pagination option to tab footer
- update `RecentTxCard` component to pass options for NUMBER_OF_TX_PER_PAGE, TRUNCATE_LENGTH, and Pagination type so RecentTxCard can be reused on the home page and the transactions page.

https://user-images.githubusercontent.com/8676844/179129273-1a5ee541-74da-4745-b23c-491b02fdf513.mov

### CODE REVIEW

File: explorer/client/src/pages/transactions/Transactions.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    return (
+        <div
+            data-testid="transaction-page"
+            id="transaction"
+            className={styles.container}
+        >
+            <h1 className={styles.title}>Transactions</h1>
+            <LastestTxCard
+                count={results.count}
+                txPerPage={20}
```
Comment (666lcz) [inline explorer/client/src/pages/transactions/Transactions.tsx:83]: nit: Avoid using magic number and define a constant `const TXN_PER_PAGE = 20` at the top of the file

File: explorer/client/src/pages/transactions/Transactions.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                paginationtype="pagination"
+                truncateLength={45}
```
Comment (666lcz) [inline explorer/client/src/pages/transactions/Transactions.tsx:85]: ditto

File: explorer/client/src/components/transaction-card/RecentTxCard.tsx [hunk 3]
```diff
@@ -236,12 +249,25 @@ function LatestTxCardStatic() {
...
+        txPerPage = NUMBER_OF_TX_PER_PAGE,
+        truncateLength = TRUNCATE_LENGTH,
+        paginationtype = DEFAULT_PAGI_TYPE,
+    } = data;
     const [isLoaded, setIsLoaded] = useState(false);
     const [results, setResults] = useState(initState);
     const [network] = useContext(NetworkContext);
     const [searchParams] = useSearchParams();
-    const [txNumPerPage] = useState(NUMBER_OF_TX_PER_PAGE);
+    const [txNumPerPage] = useState(txPerPage);
```
[the remark this answers is not in the document]
Comment (Jibz1) [reply explorer/client/src/components/transaction-card/RecentTxCard.tsx:270]: yes we can

File: explorer/client/src/pages/transactions/Transactions.tsx (hunk 2 above)
Comment (Jibz1) [reply explorer/client/src/pages/transactions/Transactions.tsx]: done

**Comment by Jibz1** [comment]
Screen Shot 2022-07-15 at 10 12 05 AM
Screen Shot 2022-07-15 at 10 11 56 AM

File: explorer/client/src/components/transaction-card/RecentTxCard.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jibz1) [reply explorer/client/src/components/transaction-card/RecentTxCard.tsx]: done

**Review by 666lcz — APPROVED** [review]

**Pull request merged**